<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logindlgClass</class>
 <widget class="QDialog" name="logindlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>150</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Identification - APP</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ID_ST_USERNAME">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="username"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ID_ST_PASSWORD">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">Password:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QFrame" name="frLanguage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ID_ST_LANGUAGE">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">Language:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="languagename"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QFrame" name="frArea">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ID_ST_AREA">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">Area:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="areaname"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="cboWindowsLogin">
       <property name="text">
        <string>Use Windows Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="okBt">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBt">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>okBt</tabstop>
  <tabstop>cancelBt</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>cboWindowsLogin</tabstop>
  <tabstop>areaname</tabstop>
  <tabstop>languagename</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
